### PR TITLE
Added matchfile

### PR DESCRIPTION
**Pull request by tclaus**

### CODE REVIEW

File: Matchfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+git_url("https://github.com/jhass/insporation-certificates")
```
Comment (jhass) [inline Matchfile:1]: Maybe better `git@github.com:jhass/insporation-certificates.git`?

File: Matchfile [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+storage_mode("git")
+
+type("development") # The default type, can be: appstore, adhoc, enterprise or development
+
+app_identifier(["eu.jhass.insporation"])
+username("thorstenclaus@web.de") # Your Apple Developer Portal username
```
Comment (jhass) [inline Matchfile:8]: Should we really set this here? Given we don't really have a shared account, maybe better to let every user set it?

File: Matchfile (hunk 1 above)
Comment (tclaus) [reply Matchfile:1]: Don't know if this 'better' or follows a recommended way - the URL was added by a generator. Change it

File: Matchfile (hunk 2 above)
Comment (tclaus) [reply Matchfile:8]: Nope - Should use the shared developer account; During my local tests - this failed.

**Pull request merged**